lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/google.rb

```diff
@@ -358,7 +364,11 @@ def create_disk(name, size_gb, zone_name, source_image)
       existing_size_gb = disk.size_gb.nil? ? nil : disk.size_gb.to_i
       existing_zone_name = disk.zone_name.nil? ? nil : disk.zone_name.split('/').last
       existing_source_image = disk.source_image.nil? ? nil : disk.source_image.split('/').last
-      if size_gb == existing_size_gb && zone_name == existing_zone_name && source_image == existing_source_image
+      existing_type = disk.type.nil? ? nil : disk.type.split('/').last
+      if size_gb == existing_size_gb &&
+         zone_name == existing_zone_name &&
+         source_image == existing_source_image &&
+         type == existing_type
```

Comment:
Ruby style guide says that continued lines should indent to the first element, not the traditional 2-line indent.